website/docs/r/is_share_mount_target.html.markdown

```diff
@@ -126,7 +151,9 @@ The following arguments are supported:
 
 The following attributes are exported:
 
-
+- `allow_ip_spoofing` - (Bool) Indicates whether source IP spoofing is allowed on this interface. If false, source IP spoofing is prevented on this interface. If true, source IP spoofing is allowed on this interface.
+  - `auto_delete` - (Bool) Indicates whether this virtual network interface will be automatically deleted when target is deleted
+  - `enable_infrastructure_nat` - (Bool) If `true`:- The VPC infrastructure performs any needed NAT operations.- `floating_ips` must not have more than one floating IP.If `false`:- Packets are passed unchanged to/from the network interface,  allowing the workload to perform any needed NAT operations.- `allow_ip_spoofing` must be `false`.- If the virtual network interface is attached:  - The target `resource_type` must be `bare_metal_server_network_attachment`.  - The target `interface_type` must not be `hipersocket`.
```

Comment:
data source documentation?

Reply:
share mount target GET response has only virtual_network_interface reference which is already added.
nothing new in response of mount target for Phase 2

Reply:
added some missing items